ui/app/AppMain.qml

```diff
@@ -347,177 +332,197 @@ Item {
             }
         }
 
-        appView: StackLayout {
-            id: appView
+        appView: ColumnLayout {
             anchors.fill: parent
-            currentIndex: {
-                if(mainModule.activeSection.sectionType === Constants.appSection.chat) {
-                    return Constants.appViewStackIndex.chat
+
+            ModuleWarning {
```

Comment:
`ModuleWarning` logically some kind of `Module` or `Modal`?

Reply:
It's kinda like a banner actually. I can rename it if you want